{"hierarchy":{"paths":[["doc:\/\/com.cloudx.sdk.core\/documentation\/CloudXCore","doc:\/\/com.cloudx.sdk.core\/documentation\/CloudXCore\/CloudXError","doc:\/\/com.cloudx.sdk.core\/documentation\/CloudXCore\/CloudXError\/LocalizedError-Implementations"]]},"sections":[],"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"var"},{"text":" ","kind":"text"},{"kind":"identifier","text":"errorDescription"},{"kind":"text","text":": "},{"text":"String","preciseIdentifier":"s:SS","kind":"typeIdentifier"},{"text":"? { ","kind":"text"},{"kind":"keyword","text":"get"},{"text":" }","kind":"text"}],"platforms":["iOS"],"languages":["swift"]}]}],"metadata":{"modules":[{"name":"CloudXCore"}],"roleHeading":"Instance Property","fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"errorDescription"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"?"}],"symbolKind":"property","extendedModule":"CloudXCore","externalID":"s:10CloudXCore0A6XErrorO16errorDescriptionSSSgvp","title":"errorDescription","role":"symbol"},"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/com.cloudx.sdk.core\/documentation\/CloudXCore\/CloudXError\/errorDescription"},"abstract":[{"type":"text","text":"A localized message describing what error occurred."}],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/cloudxcore\/cloudxerror\/errordescription"]}],"kind":"symbol","schemaVersion":{"patch":0,"major":0,"minor":3},"references":{"doc://com.cloudx.sdk.core/documentation/CloudXCore":{"type":"topic","abstract":[],"title":"CloudXCore","role":"collection","kind":"symbol","identifier":"doc:\/\/com.cloudx.sdk.core\/documentation\/CloudXCore","url":"\/documentation\/cloudxcore"},"doc://com.cloudx.sdk.core/documentation/CloudXCore/CloudXError":{"url":"\/documentation\/cloudxcore\/cloudxerror","role":"symbol","kind":"symbol","title":"CloudXError","fragments":[{"text":"enum","kind":"keyword"},{"kind":"text","text":" "},{"text":"CloudXError","kind":"identifier"}],"type":"topic","abstract":[{"text":"An error type that represents the CloudX SDK errors.","type":"text"}],"navigatorTitle":[{"text":"CloudXError","kind":"identifier"}],"identifier":"doc:\/\/com.cloudx.sdk.core\/documentation\/CloudXCore\/CloudXError"},"doc://com.cloudx.sdk.core/documentation/CloudXCore/CloudXError/LocalizedError-Implementations":{"identifier":"doc:\/\/com.cloudx.sdk.core\/documentation\/CloudXCore\/CloudXError\/LocalizedError-Implementations","type":"topic","abstract":[],"title":"LocalizedError Implementations","url":"\/documentation\/cloudxcore\/cloudxerror\/localizederror-implementations","role":"collectionGroup","kind":"article"},"doc://com.cloudx.sdk.core/documentation/CloudXCore/CloudXError/errorDescription":{"role":"symbol","title":"errorDescription","identifier":"doc:\/\/com.cloudx.sdk.core\/documentation\/CloudXCore\/CloudXError\/errorDescription","abstract":[{"type":"text","text":"A localized message describing what error occurred."}],"url":"\/documentation\/cloudxcore\/cloudxerror\/errordescription","kind":"symbol","type":"topic","fragments":[{"text":"var","kind":"keyword"},{"kind":"text","text":" "},{"text":"errorDescription","kind":"identifier"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"text":"?","kind":"text"}]}}}